editor/theme/templates/west-referral/invitation/page.tsx

```
@@ -300,32 +300,17 @@ export default function InvitationPageTemplate({
                 {
-                  href: "https://www.polestar.com/kr/legal/privacy-policy/",
-                  text: "개인정보 처리방침",
-                },
-                {
-                  href: "https://www.polestar.com/kr/legal/terms-and-conditions/",
-                  text: "이용약관",
-                },
-                {
-                  href: "https://www.polestar.com/kr/legal/terms-and-conditions/#terms-of-use-for-test-drive",
-                  text: "시승 이용약관",
+                  href: design.footer.link_privacy,
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)